### DISCUSSION TITLE
Did the troubleshooting wizard break?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3810] [Q&A Feedback]
I wanted to look at the troubleshooting wizard to see what problems it was handling, but on 1.13.5 I noticed that the lists are empty and once I loaded the wizard in German (which contains strings like in the buttons that are not translated), the wizard stayed German, after closing it, and switching the language. It required me to restart Sandman to have the troubleshooting wizard to be in English.

![wizard](https://github.com/sandboxie-plus/Sandboxie/assets/1449429/0d7bd693-7337-44e8-8d53-1041c170f6c3)
![wizard2ndpage](https://github.com/sandboxie-plus/Sandboxie/assets/1449429/b9406d3e-73cb-436f-9de0-4a769833ddaa)
![wizardtroublebrowser](https://github.com/sandboxie-plus/Sandboxie/assets/1449429/53219ee0-0c2b-4ad5-bd21-93e1103868ae)

Since I don't need the troubleshooting wizard, normally, I did not look at it. Is it still working for you?

**Comment by bastik-1001** [comment]
This appears to be a regression, as 1.13.3 and 1.13.4 show the entries. Also, 1.13.5 only shows "Fix issues with sandboxing", while 1.13.4 shows that and a button to fix issues with the UI.

**Reply by offhub** [reply]
It seems to be an issue with the Qt version.

![sbie3810_qtdiff](https://github.com/sandboxie-plus/Sandboxie/assets/6871698/03519a10-4593-4b58-995b-061aade230a1)

**Reply by bastik-1001** [reply]
Nice that you figured out what is causing it.

**Comment by DavidXanatos** [comment]
Yes it broke and its due to the new Qt Version: https://github.com/sandboxie-plus/Sandboxie/pull/3694
when building sbie+ with 5.15.2 all works fine @LumitoLuma this issue needs taking looking to

**Reply by LumitoLuma** [reply]
Updater doesn't seem to work either. However, it's very strange, since LTS builds should keep feature compatibility between versions and only fix bugs and security issues...

**Reply by DavidXanatos** [reply]
it seams the issue is related to the QAbstractFileEngine, our implementation of C7zFileEngine was a bit lazy and with the new qt it broke, now lets see whats wrong with the updater :/

**Reply by bastik-1001** [reply]
About the updater, yes, the update page in the settings does not list the current or latest version, nor the insider build. When I manually check for updates a message appears that it failed as the server could not be reached.

**Reply by DavidXanatos** [reply]
the updater issue seams to be SSL related eider we lack the right version of worse a flag was missing for the qt build and it needs to be re build with ssl enabled

**Reply by DavidXanatos** [reply]
yes C:\Qt\Qt5.15.13\msvc2019_64\include\QtNetwork\qtnetwork-config.h contains #define QT_NO_OPENSSL true
@LumitoLuma could you pelase updaet the build script to include openssl support in the Qt build

**Reply by LumitoLuma** [reply]
Sure, I'll update them.

**Reply by DavidXanatos** [reply]
here is the full list of discrepancies

![grafik](https://github.com/sandboxie-plus/Sandboxie/assets/3890945/adb20087-ced4-4462-b272-49ba2eff93e4)


the left side is what it shoudl be the right side is what it is

**Reply by LumitoLuma** [reply]
Qt configure also shows:

```
OpenSSL 1.1 ............................ no
```

Maybe I could use Qt's openssl for that.

**Reply by LumitoLuma** [reply]
Qt build with openssl seemed to succeed. Going to test building SBIE against it.

**Reply by LumitoLuma** [reply]
@DavidXanatos It works! Please sync your qt-builds fork and update the SBIE Plus build script (install_qt.cmd). I ended up using OpenSSL 1.1.1p, so no additional changes to the build workflow are needed.

**Reply by DavidXanatos** [reply]
@LumitoLuma seams working great, thanks :D
